<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modificar_Sumi</class>
 <widget class="QDialog" name="Modificar_Sumi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar suministros</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="4">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_tipo">
         <property name="text">
          <string>Tipo</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="tipo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_fecha">
         <property name="text">
          <string>Fecha de Llegada</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fecha"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_stock">
         <property name="text">
          <string>Stock</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="stock"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_pais">
         <property name="text">
          <string>Pais/Procedencia</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="pais">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_dosis">
         <property name="text">
          <string>Dosis/P</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="dosis">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_calidad">
         <property name="text">
          <string>Calidad</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="calidad">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="vacuna">
       <property name="text">
        <string>Vacuna</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="prueba">
       <property name="text">
        <string>Prueba</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="mascarilla">
       <property name="text">
        <string>Mascarilla</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="medicamento">
       <property name="text">
        <string>Medicamento</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="enviarpushbuton">
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
